advocacy_docs/supported-open-source/pgbackrest/08-multiple-repositories.mdx

```diff
@@ -0,0 +1,586 @@
+
+```bash
+root# systemctl stop edb-as-13.service
+$ mv /var/lib/edb/as13/data /var/lib/edb/as13/data.orig
+$ mkdir -m 700 /var/lib/edb/as13/data
+
+$ pgbackrest restore --stanza=demo --target="2021-04-19 14:38:55.313716+00:00" --type=time --no-delta --target-action=promote
+P00   INFO: restore command begin ...
+P00   INFO: write updated /var/lib/edb/as13/data/postgresql.auto.conf
+P00   INFO: restore global/pg_control (performed last to ensure aborted restores cannot be started)
+P00   INFO: restore command end: completed successfully
+
+$ cat /var/lib/edb/as13/data/postgresql.auto.conf|grep restore_command
+restore_command = 'pgbackrest --stanza=demo archive-get %f "%p"'
+
+root# systemctl start edb-as-13.service
+$ cat /var/lib/edb/as13/data/log/edb-2021-04-19_145147.log
+...
+LOG:  starting point-in-time recovery to 2021-04-19 14:38:55.313716+00
+P00   INFO: found 000000010000000000000018 in the repo2: 13-1 archive
+P00   INFO: found 000000010000000000000019 in the repo2: 13-1 archive
+LOG:  recovery stopping before commit of transaction 1183, time 2021-04-19 14:38:55.314118+00
+LOG:  redo done at 0/19000100
+P00   INFO: unable to find 00000002.history in the archive
+LOG:  selected new timeline ID: 2
+LOG:  archive recovery complete
+P00   INFO: pushed WAL file '00000002.history' to the archive
+
+$ ls /var/lib/edb/as13/backups/repo*/archive/demo/13-1/00000002.history 
+/var/lib/edb/as13/backups/repo1/archive/demo/13-1/00000002.history
+/var/lib/edb/as13/backups/repo2/archive/demo/13-1/00000002.history
+```
+
+PostgreSQL found the WALs needed for recovery in our **repo2**, picked a new timeline and pushed the history file to the repositories. 
+
+```bash
+root# systemctl stop edb-as-13.service
+$ rm -rf /var/lib/edb/as13/data/*
+$ rm -rf /var/lib/edb/as13/backups/repo1/archive/demo/13-1/00000002.history
+$ pgbackrest restore --stanza=demo --target="2021-04-19 14:38:55.313716+00:00" --type=time --target-timeline=current --no-delta --target-action=promote
+P00   INFO: restore command begin ...
+P00   INFO: repo2: restore backup set 20210419-143749F
+P00   INFO: write updated /var/lib/edb/as13/data/postgresql.auto.conf
+P00   INFO: restore global/pg_control (performed last to ensure aborted restores cannot be started)
+P00   INFO: restore command end: completed successfully
+
+root# systemctl start edb-as-13.service
+$ cat /var/lib/edb/as13/data/log/edb-2021-04-19_145721.log
+P00   INFO: found 00000002.history in the repo2: 13-1 archive
+P00   INFO: unable to find 00000003.history in the archive
+LOG:  selected new timeline ID: 3
+P00   INFO: pushed WAL file '00000003.history' to the archive
+```
+
+Let's retry it by moving the history file in **repo1** now.
```

Comment:
You should better describe the issue that is occurring here (and that it is an issue), before mentioning moving the history file (first describe the issue, then how to solve it).

Reply:
New text proposal in b234122 to describe what timeline history files are and why they are necessary.

Reply:
Much better, thanks.
I added one comment replacing 'Every time' with 'Whenever'.

Reply:
Fixed in 7fa8ff1